3rdparty/CMakeLists.txt

```diff
@@ -33,25 +33,22 @@ ExternalProject_Add(udis86_ext
   URL ${UDIS86_URL} # udis86 source directory, gets copied to binary dir in out-of-source build
   SOURCE_DIR ${UDIS86_SOURCE_DIR}
   BUILD_IN_SOURCE TRUE # in out-of-source build "source" is in binary dir, so it'll be performed there
-  CONFIGURE_COMMAND
-    COMMAND <SOURCE_DIR>/autogen.sh
-    COMMAND <SOURCE_DIR>/configure CC=${CMAKE_C_COMPILER}
-            --without-yasm --without-sphinx-build
+  CMAKE_ARGS -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
   INSTALL_COMMAND cmake -E echo "Skipping install step"
 )
 
 ExternalProject_Get_Property(udis86_ext SOURCE_DIR)
 
 # NB! Enlist files explicitly (no globbing) so that cmake generates a valid
 # file list for the linker before udis86 is built
-set(UDIS86_OBJ_DIR ${SOURCE_DIR}/libudis86/.libs)
+set(UDIS86_OBJ_DIR ${SOURCE_DIR}/libudis86/CMakeFiles/libudis86.dir)
```

Comment:
This seems very CMake version dependent... CMake doesn't specify where obj files will be put. Previously we could rely on it based on how autoconf script behaved, but now - I'm not so sure.

It seems like we get libudis86 target somehow now. Can we use `$<TARGET_OBJECTS:libudis86>` generator expression instead? So, something like this:

```cmake
add_subdirectory(udis86)
set_target_properties(udis86 PROPERTIES
  IMPORTED_OBJECTS $<TARGET_OBJECTS:libudis86>
)
```

Reply:
Fixed.